vip-helpers/vip-clean-term-cache.php

```diff
@@ -52,8 +54,17 @@ public function edited_term_taxonomy_action( $tt_id, $taxonomy ) {
 		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
 		$objects = $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->term_relationships WHERE term_taxonomy_id = %d", $tt_id ) );
 
+		$num_objects = intval( $objects );
+
+		// Allows admin users to see saved changes in the edit terms page, when editing a topic
+		// Short circuits scheduling and runs the method synchronously
+		if ( ( $num_objects < $this->admin_limit ) && is_admin() && ! wp_doing_ajax() ) {
+			$this->cron_action( $tt_id, $taxonomy );
```

Comment:
I don't think we need to run the cron action here - would just an early return like in the conditional below be sufficient?

If I understand this all correctly, this module is meant to to override some defaults if the term count is too high and instead schedule a cache purge later on. If under the "limit" it doesn't suspend cache invalidation. Here we just simply need to not suspend if it's in the admin and under the "admin limit" ?